src/components/screen/ProfileUpdate.tsx

```diff
@@ -79,6 +79,12 @@ function Screen(props: Props): React.ReactElement {
     setIsUpdating(true);
   };
 
+  const onSetting = (): void => {
+    if (props.navigation) {
```

Comment:
It'd be better to use `const navigation = useNavigation();`.

Reply:
Actually it's now unnecessary. I will remove it. Thanks.